lib/util/markdown.spec.ts

```diff
@@ -29,5 +30,13 @@ describe('util/markdown', () => {
     it('works', async () => {
       expect(await linkify(before, { repository: 'some/repo' })).toEqual(after);
     });
-  });
+  }),
+    describe('.sanitizeMarkdown', () => {
+      it('sanitizes URLs of Markdown links containing spaces to working links', async () => {
+        const before = `[Some.Package](https://foo.bar/E  F/A2B CD) ([source](https://foo.bar/EF/A2B CD/_git/Some.Package))`;
+        const after = `[Some.Package](https://foo.bar/E%20%20F/A2B%20CD) ([source](https://foo.bar/EF/A2B%20CD/_git/Some.Package))`;
```

Comment:
```suggestion
        const before = `[Some.Package](https://foo.bar/E  F/A2B CD) [Some.Package](https://foo.bar/E  F/A2B CD) ([source](https://foo.bar/EF/A2B CD/_git/Some.Package))`;
        const after = `[Some.Package](https://foo.bar/E%20%20F/A2B%20CD) [Some.Package](https://foo.bar/E%20%20F/A2B%20CD) ([source](https://foo.bar/EF/A2B%20CD/_git/Some.Package))`;
```

Should also replace same link multiple times